packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.ts

```diff
@@ -52,6 +52,19 @@ export class CollectionsInfoOverlayComponent implements OnInit {
     this.collectionColumns = Object.keys(this.showingCollection[0]).filter(
       (column) => !['uuid', 'hits', 'isCut'].includes(column) // FIXME - this is an ugly hack. But currently hits from tracks make track collections unusable. Better to have exlusion list passed in.
     );
+
+    this.collectionColumns = this.collectionColumns.map((column) => {
+      return {
+        name: column,
+        isNumber: typeof this.showingCollection[0][column] === 'number',
+      };
+    });
+  }
+
+  sort(columnName, order) {
+    if (order === 'asc')
+      this.showingCollection.sort((a, b) => a[columnName] - b[columnName]);
+    else this.showingCollection.sort((a, b) => b[columnName] - a[columnName]);
```

Comment:
Could be improved to:

```ts
sort(column: string) {
  if (order === 'asc') {
    this.showingCollection.sort((a, b) => a < b ? - 1 : 1);
  } else {
    this.showingCollection.sort((a, b) => a > b ? - 1 : 1);
  }
}
```

This will also sort string and other types of values.